src/components/generics/Markdown.vue

```diff
@@ -321,4 +321,13 @@ div[c2c\:role='video'] {
     }
   }
 }
+
+@media screen and(max-width: 844px) {
```

Comment:
Where does this value `884` come from? We use bulma breakpoints to distinguish between phone, tablet, desktop and widescreen (768, 1023, 1215 ,1407, +- 1px depending on comparison made)

Reply:
When I put $phone on my doc, it has got an error and I wasn't able to do my internship mission correctly.In school, I learnt to add some value in this case. On the inspector,it was written this value, I wrote this value and the error disappeared.